engine/flink/management/periodic/src/main/scala/pl/touk/nussknacker/engine/management/periodic/Utils.scala

```diff
@@ -6,13 +6,15 @@ import com.typesafe.scalalogging.LazyLogging
 
 import java.util.concurrent.TimeoutException
 import scala.concurrent.duration.DurationInt
-import scala.concurrent.{Await, Future}
+import scala.concurrent.{Await, ExecutionContext, Future}
+import scala.util.{Failure, Success}
 
 object Utils extends LazyLogging {
 
-  private val GracefulStopTimeout         = 5 seconds
-  private val ActorResolutionTimeout      = 5 seconds
-  private val TotalActorResolutionTimeout = 16 seconds
+  private val GracefulStopTimeout    = 5 seconds
+  private val ActorResolutionTimeout = 5 seconds
+  private val ActorResolutionPause   = 500 milliseconds
```

Comment:
Shouldn't this be more like 10-50ms (and more retries to match shorter sleep time)?